services/worker/src/worker/job_runner.py

```diff
@@ -298,8 +298,8 @@ class JobRunner(ABC):
 
     Args:
         job_info (:obj:`JobInfo`):
-            The job to process. It contains the job_id, the job type, the dataset, the config, the split
-            the force flag, and the priority level.
+            The job to process. It contains the job_id, the job type, the dataset, the config, the split,
```

Comment:
I think all these changes should go now to job_manager.py file

Reply:
yes, the rebase was big :)